service_management/app/repositories/analysis/dao/Tables.scala

```diff
@@ -304,7 +309,7 @@ trait Tables extends HasDatabaseConfigProvider[JdbcProfile] with ColumnTypeMappe
       maybeTuple: Option[ResultRow]
   ): Option[AnalysisResult] =
     maybeTuple.flatMap(fromResultRow)
-
+  //todo add externalIdSource, treatment, residualMaterial, description
```

Comment:
I have a question concerning different actors (persons) with different roles and dates connected to a sample (i.e. the sampleCreation [Prøveuttak]). From frontend we have an array with person-, role- and date objects and. How should we send this to backend? An array called Persons, where each entry is an object with type { name: string, role: string, date: string}?